config.toml

```diff
@@ -55,16 +55,22 @@ weight = 1
 hasChildren = true
 
     [[menu.main]]
+    parent = "About"
     name = "About Us"
     URL = "about"
-    parent = "About"
     weight = 1
 
     [[menu.main]]
     parent = "About"
     name = "methodology"
     URL = "methodology"
     weight = 2
+    
+    [[menu.main]]
```

Comment:
this will technically work, but we don't have a page for it to point to.

Reply:
How do I _make_ a page for it to point to?

Reply:
😸